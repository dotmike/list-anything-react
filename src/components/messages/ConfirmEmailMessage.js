import React from "react";
import { Message } from "semantic-ui-react";

const ConfirmEmailMessage = () => (
  <Message info style={{ textAlign: "center" }}>
    <Message.Header>
      Please, verify your email to unlock awesomeness
    </Message.Header>
  </Message>
);

export default ConfirmEmailMessage;
