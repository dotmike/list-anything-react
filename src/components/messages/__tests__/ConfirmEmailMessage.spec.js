import React from "react";
import { shallow } from "enzyme";

import ConfirmEmailMessage from "../ConfirmEmailMessage";

describe("ConfirmEmailMessage", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<ConfirmEmailMessage />);
    expect(wrapper).toMatchSnapshot();
  });
});
