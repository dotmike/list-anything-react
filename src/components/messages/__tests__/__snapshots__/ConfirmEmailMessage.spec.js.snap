// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmEmailMessage renders correctly 1`] = `
<Message
  info={true}
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <MessageHeader>
    Please, verify your email to unlock awesomeness
  </MessageHeader>
</Message>
`;
